store/sqlstore/command_store.go

```diff
@@ -139,19 +184,20 @@ func (s SqlCommandStore) Update(cmd *model.Command) (*model.Command, *model.AppE
 }
 
 func (s SqlCommandStore) AnalyticsCommandCount(teamId string) (int64, *model.AppError) {
-	query :=
-		`SELECT
-			COUNT(*)
-		FROM
-			Commands
-		WHERE
-			DeleteAt = 0`
+	query := s.getQueryBuilder().
+		Select("COUNT(*)").
+		From("Commands").
+		Where(sq.Eq{"DeleteAt": 0})
 
 	if len(teamId) > 0 {
-		query += " AND TeamId = :TeamId"
+		query = query.Where(sq.Eq{"TeamId": teamId})
+	}
+	queryStr, args, err := query.ToSql()
```

Comment:
nit: missing new line

Reply:
also can we use the same naming convention as above

```
sql, args, err := query.ToSql()
```

Reply:
Done.